index.html

```diff
@@ -238,13 +260,15 @@ <h4>Skills</h4>
                                     alt="Github-icon"></a></li>
                         <li><a href="#"><img src="./ImageP1/angellist@.png" alt="Angellist-icon"></a></li>
                     </ul>
```

Comment:
- [ ] Kindly position the social media icons to be at the center of the page instead of aligning them to the left side of the page. This would make your design look identical to that of the Figma template that you chose. I would advise that you use the CSS flexbox to make this section responsive. 👍 😎 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/228956460-fd025477-c5ee-4e22-91fd-e3f3561f0bd7.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/228956565-3a514147-10f2-4e90-b5eb-fb0755fbdf6d.png)